scripts/install/opt/seagate/cortx/motr/conf/support.yaml

```diff
@@ -0,0 +1,4 @@
+
+bundle:
+    create:
+      url: "sh /opt/seagate/cortx/motr/libexec/m0reportbug-bundler"
```

Comment:
@sachinpunadikar  @papan-singh  : Do we need to pass the directory from where it wants to create the files or it is taken care in support bundle framework.
By default, it will use /tmp folder to create support bundle outputs if not arguments are passed

Reply:
we  can find generated support bundle inside
/var/log/seagate/support_bundle/  - actual path and
/tmp/support_bundle/ - symlink path
No need to pass any arguments, path created internally.

`[root@ssc-vm-5431 ~]# cortxcli support_bundle generate --help
usage: cortxcli support_bundle generate [-h] [--os]
                                        [-c {all,csm,sspl,s3server,motr,hare,provisioner,os,health_map,manifest,alerts,uds,elasticsearch,ha} [{all,csm,sspl,s3server,motr,hare,provisioner,os,health_map,manifest,alerts,uds,elasticsearch,ha} ...]]
                                        comment

positional arguments:
  comment               Specify the Reason for Generating Support Bundle.

optional arguments:
  -h, --help            show this help message and exit
  --os                  Generate Only OS Logs
  -c {all,csm,sspl,s3server,motr,hare,provisioner,os,health_map,manifest,alerts,uds,elasticsearch,ha} [{all,csm,sspl,s3server,motr,hare,provisioner,os,health_map,manifest,alerts,uds,elasticsearch,ha} ...]
                        Generate Bundle for Specific components defined.
`